### DISCUSSION TITLE
Unable to remotely connect to native desktop of Ubuntu18.04 in Windows

### DISCUSSION THREAD

**Discussion by ICanFlyGFC** [neutrinolabs/xrdp#2019] [Q&A]
Hello! When I try to connect with Ubuntu I want to see a native desktop. But I get a simple desktop which is not actually one.
I get 
![image](https://user-images.githubusercontent.com/50896307/137420327-efa05839-be4a-4d70-9717-14ce5292291a.png)
But my actual desktop is
![image](https://user-images.githubusercontent.com/50896307/137420488-32fc1390-497b-4047-b00d-a3ff79916c5c.png)
How can I solve this problem? Thanks!

**Comment by matt335672** [comment]
Hi @ICanFlyGFC.

I'm a little confused here.

That top picture looks like a CentOS 7 backdrop to me. Here's a shot of my CentOS 7 VM:-

![c7](https://user-images.githubusercontent.com/30179339/137465245-501034c3-7658-469d-a299-e82f7d677ea9.png)

I didn't think that Ubuntu shipped with CentOS backgrounds.

I'm sorry to have to ask this, and please don't be offended, but can we first make sure that you're connecting to the same machine? What's the output of `uname -a` on both desktops?

Assuming you are connecting to the same machine have a look at #1723 which may well answer your question.
